nbodykit/base/mesh.py

```diff
@@ -359,3 +371,28 @@ def save(self, output, dataset='Field', mode='real'):
                             bb.attrs[key] = json_str
                         except:
                             warnings.warn("attribute %s of type %s is unsupported and lost while saving MeshSource" % (key, type(value)))
+
+
+def initialize_pm(self):
```

Comment:
This is not a member of the class?

Reply:
It's not right now but it could be. But this is part of the issues I was having with MeshSource. Ideally, we should probably have some sort of a mix-in MeshSource class  to CatalogMesh that doesn't need to be initialized, but we need to work around the initializing ``pm`` and using the vector forms of ``Nmesh`` and ``BoxSize`` from the ``pm`` attribute